ui/src/api/generated/api.ts

```diff
@@ -9971,7 +9971,7 @@ export const UserApiFp = function(configuration?: Configuration) {
          * @param {*} [options] Override http request option.
          * @throws {RequiredError}
          */
-        async login(loginInfo: ParamLoginParam, options?: RawAxiosRequestConfig): Promise<(axios?: AxiosInstance, basePath?: string) => AxiosPromise<ResponseAPIResponse>> {
+        async login(loginInfo: ParamLoginParam, options?: RawAxiosRequestConfig): Promise<(axios?: AxiosInstance, basePath?: string) => Promise<ResponseAPIResponse>> {
```

Comment:
I adjusted the format of the request return and now the returned type is no longer AxiosPromise.

Reply:
The reason for adjusting the format is because I hope to directly obtain the data at the business layer.
![image](https://github.com/oceanbase/ob-operator/assets/62841398/de93cf13-3520-4ea9-8880-35b542bcae31)

Reply:
But in this `api.ts` there are other `AxiosPromise` returned from API methods. If you decide to modify the return type to `Promise`, you'd better modify all of methods.

Reply:
Yes, I will modify all methods in the next PR.